forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -1193,7 +1193,11 @@ public static PlayerCollection getDefinedPlayers(final Card card, final String d
         } else if (defined.equals("You")) {
             players.add(player);
         } else if (defined.equals("Opponent")) {
-            players.addAll(player.getOpponents());
+            if (sa.hasParam("InTurnOrder") && sa.getParam("InTurnOrder").equals("True")) {
```

Comment:
this is a good catch but will require a wider approach imo as this wording seems to only emphasize the default anyway, e.g.:
> As Dredge the Mire resolves, first the next opponent in turn order (or, if it’s an opponent’s turn, the opponent whose turn it is) chooses a creature card in their graveyard, then each other opponent in turn order does the same knowing the choices made before them. Then all the chosen cards are returned at the same time.

Reply:
That is a very good catch. I did not like adding this InTurnOrder check to only one place (Opponent). I'll look into it more.

Reply:
- mb something like this to align the order instead?
```
// try sort in APNAP order
PlayerCollection sortedUnique = sa.getHostCard().getGame().getPlayersInTurnOrder(sa.getHostCard().getGame().getPhaseHandler().getPlayerTurn());
sortedUnique.retainAll(resultUnique);
return sortedUnique;
```
- the Opponent ordering in AbilityUtils might still be updated with the new default since quite a few places only call out to `getDefinedPlayers()` so a few corner cases could probably continue to hide for future cleanup

Reply:
@tool4ever I created a new commit that reverted all of the `InTurnOrder `stuff. I went with changing `StartingWithActivator ` to `StartingWith` and that new parameter can take `You `or `Oppponent`. If you would like me to create a new PR for these changes and remove this PR I can do so. Please see [commit](https://github.com/Card-Forge/forge/pull/6226/commits/b7e4640ba598d11c8762b3f76c34d69df74050b8).

Reply:
that's fine, we usually squash merge if needed